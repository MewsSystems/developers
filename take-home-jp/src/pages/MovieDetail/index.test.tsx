import React from 'react';
import renderer from 'react-test-renderer';
import MoveDetail from './';

it('renders correctly', () => {
  const tree = renderer.create(<MoveDetail />).toJSON();
  expect(tree).toMatchSnapshot();
});
