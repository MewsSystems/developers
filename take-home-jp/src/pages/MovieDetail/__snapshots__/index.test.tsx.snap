// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  style={
    Object {
      "backgroundImage": "url('https://image.tmdb.org/t/p/originalundefined')",
      "backgroundPositionX": "center",
      "backgroundPositionY": "center",
      "backgroundRepeat": "no-repeat",
      "backgroundSize": "cover",
      "display": "flex",
      "flex": "1",
      "minHeight": "100vh",
      "textAlign": "left",
    }
  }
>
  <div
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.8)",
        "display": "flex",
        "flex": 1,
        "justifyContent": "center",
        "width": "100%",
      }
    }
  >
    <div
      className="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
      style={
        Object {
          "display": "flex",
          "flexDirection": "column",
          "justifyContent": "center",
        }
      }
    >
      <div
        className="MuiBox-root css-8qb8m4"
        style={
          Object {
            "display": "flex",
            "justifyContent": "space-between",
          }
        }
      >
        <h1
          className="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
          style={
            Object {
              "flex": 3,
              "fontSize": "3.5rem",
              "fontWeight": "800",
            }
          }
        />
        <div
          className="MuiBox-root css-0"
          style={
            Object {
              "display": "flex",
              "flex": 1,
              "flexDirection": "column",
              "justifyContent": "flex-start",
            }
          }
        >
          <legend
            className="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
          >
            <strong>
              Average Rating
            </strong>
          </legend>
          <span
            aria-label="0 Stars"
            className="MuiRating-root MuiRating-sizeSmall Mui-readOnly MuiRating-readOnly css-1nbsiwm-MuiRating-root"
            onMouseLeave={[Function]}
            onMouseMove={[Function]}
            role="img"
          >
            <span
              className="MuiRating-decimal css-34he1w-MuiRating-decimal"
            >
              <span
                style={
                  Object {
                    "overflow": "hidden",
                    "position": "absolute",
                    "width": "0%",
                  }
                }
              >
                <span
                  className="MuiRating-icon MuiRating-iconEmpty css-1c99szj-MuiRating-icon"
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                    data-testid="StarBorderIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M22 9.24l-7.19-.62L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21 12 17.27 18.18 21l-1.63-7.03L22 9.24zM12 15.4l-3.76 2.27 1-4.28-3.32-2.88 4.38-.38L12 6.1l1.71 4.04 4.38.38-3.32 2.88 1 4.28L12 15.4z"
                    />
                  </svg>
                </span>
              </span>
              <span
                style={Object {}}
              >
                <span
                  className="MuiRating-icon MuiRating-iconEmpty css-1c99szj-MuiRating-icon"
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                    data-testid="StarBorderIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M22 9.24l-7.19-.62L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21 12 17.27 18.18 21l-1.63-7.03L22 9.24zM12 15.4l-3.76 2.27 1-4.28-3.32-2.88 4.38-.38L12 6.1l1.71 4.04 4.38.38-3.32 2.88 1 4.28L12 15.4z"
                    />
                  </svg>
                </span>
              </span>
            </span>
            <span
              className="MuiRating-decimal css-34he1w-MuiRating-decimal"
            >
              <span
                style={
                  Object {
                    "overflow": "hidden",
                    "position": "absolute",
                    "width": "0%",
                  }
                }
              >
                <span
                  className="MuiRating-icon MuiRating-iconEmpty css-1c99szj-MuiRating-icon"
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                    data-testid="StarBorderIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M22 9.24l-7.19-.62L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21 12 17.27 18.18 21l-1.63-7.03L22 9.24zM12 15.4l-3.76 2.27 1-4.28-3.32-2.88 4.38-.38L12 6.1l1.71 4.04 4.38.38-3.32 2.88 1 4.28L12 15.4z"
                    />
                  </svg>
                </span>
              </span>
              <span
                style={Object {}}
              >
                <span
                  className="MuiRating-icon MuiRating-iconEmpty css-1c99szj-MuiRating-icon"
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                    data-testid="StarBorderIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M22 9.24l-7.19-.62L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21 12 17.27 18.18 21l-1.63-7.03L22 9.24zM12 15.4l-3.76 2.27 1-4.28-3.32-2.88 4.38-.38L12 6.1l1.71 4.04 4.38.38-3.32 2.88 1 4.28L12 15.4z"
                    />
                  </svg>
                </span>
              </span>
            </span>
            <span
              className="MuiRating-decimal css-34he1w-MuiRating-decimal"
            >
              <span
                style={
                  Object {
                    "overflow": "hidden",
                    "position": "absolute",
                    "width": "0%",
                  }
                }
              >
                <span
                  className="MuiRating-icon MuiRating-iconEmpty css-1c99szj-MuiRating-icon"
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                    data-testid="StarBorderIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M22 9.24l-7.19-.62L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21 12 17.27 18.18 21l-1.63-7.03L22 9.24zM12 15.4l-3.76 2.27 1-4.28-3.32-2.88 4.38-.38L12 6.1l1.71 4.04 4.38.38-3.32 2.88 1 4.28L12 15.4z"
                    />
                  </svg>
                </span>
              </span>
              <span
                style={Object {}}
              >
                <span
                  className="MuiRating-icon MuiRating-iconEmpty css-1c99szj-MuiRating-icon"
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                    data-testid="StarBorderIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M22 9.24l-7.19-.62L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21 12 17.27 18.18 21l-1.63-7.03L22 9.24zM12 15.4l-3.76 2.27 1-4.28-3.32-2.88 4.38-.38L12 6.1l1.71 4.04 4.38.38-3.32 2.88 1 4.28L12 15.4z"
                    />
                  </svg>
                </span>
              </span>
            </span>
            <span
              className="MuiRating-decimal css-34he1w-MuiRating-decimal"
            >
              <span
                style={
                  Object {
                    "overflow": "hidden",
                    "position": "absolute",
                    "width": "0%",
                  }
                }
              >
                <span
                  className="MuiRating-icon MuiRating-iconEmpty css-1c99szj-MuiRating-icon"
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                    data-testid="StarBorderIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M22 9.24l-7.19-.62L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21 12 17.27 18.18 21l-1.63-7.03L22 9.24zM12 15.4l-3.76 2.27 1-4.28-3.32-2.88 4.38-.38L12 6.1l1.71 4.04 4.38.38-3.32 2.88 1 4.28L12 15.4z"
                    />
                  </svg>
                </span>
              </span>
              <span
                style={Object {}}
              >
                <span
                  className="MuiRating-icon MuiRating-iconEmpty css-1c99szj-MuiRating-icon"
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                    data-testid="StarBorderIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M22 9.24l-7.19-.62L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21 12 17.27 18.18 21l-1.63-7.03L22 9.24zM12 15.4l-3.76 2.27 1-4.28-3.32-2.88 4.38-.38L12 6.1l1.71 4.04 4.38.38-3.32 2.88 1 4.28L12 15.4z"
                    />
                  </svg>
                </span>
              </span>
            </span>
            <span
              className="MuiRating-decimal css-34he1w-MuiRating-decimal"
            >
              <span
                style={
                  Object {
                    "overflow": "hidden",
                    "position": "absolute",
                    "width": "0%",
                  }
                }
              >
                <span
                  className="MuiRating-icon MuiRating-iconEmpty css-1c99szj-MuiRating-icon"
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                    data-testid="StarBorderIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M22 9.24l-7.19-.62L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21 12 17.27 18.18 21l-1.63-7.03L22 9.24zM12 15.4l-3.76 2.27 1-4.28-3.32-2.88 4.38-.38L12 6.1l1.71 4.04 4.38.38-3.32 2.88 1 4.28L12 15.4z"
                    />
                  </svg>
                </span>
              </span>
              <span
                style={Object {}}
              >
                <span
                  className="MuiRating-icon MuiRating-iconEmpty css-1c99szj-MuiRating-icon"
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                    data-testid="StarBorderIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M22 9.24l-7.19-.62L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21 12 17.27 18.18 21l-1.63-7.03L22 9.24zM12 15.4l-3.76 2.27 1-4.28-3.32-2.88 4.38-.38L12 6.1l1.71 4.04 4.38.38-3.32 2.88 1 4.28L12 15.4z"
                    />
                  </svg>
                </span>
              </span>
            </span>
          </span>
        </div>
      </div>
      <p
        className="MuiTypography-root MuiTypography-body1 css-z2eky3-MuiTypography-root"
        style={
          Object {
            "color": "#989898",
            "fontWeight": "bold",
          }
        }
      >
         |  
      </p>
      <div
        className="MuiBox-root css-10ib5jr"
      >
        <div
          className="MuiStack-root css-niqf4j-MuiStack-root"
        />
      </div>
      <p
        className="MuiTypography-root MuiTypography-body1 css-12fp2g9-MuiTypography-root"
      />
      <div
        className="MuiBox-root css-1qm1lh"
      >
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeLarge MuiButton-containedSizeLarge MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeLarge MuiButton-containedSizeLarge css-jh47zj-MuiButtonBase-root-MuiButton-root"
          disabled={false}
          onBlur={[Function]}
          onContextMenu={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          Learn More
        </button>
      </div>
    </div>
  </div>
</div>
`;
